src/main/webapp/app/course/tutorial-groups/tutorial-groups-management/tutorial-groups-configuration/crud/create-tutorial-groups-configuration/create-tutorial-groups-configuration.component.ts

```diff
@@ -22,19 +22,19 @@ export class CreateTutorialGroupsConfigurationComponent implements OnInit, OnDes
 
     newTutorialGroupsConfiguration = new TutorialGroupsConfiguration();
     isLoading: boolean;
-    course: Course;
-
     constructor(
         private activatedRoute: ActivatedRoute,
         private router: Router,
         private tutorialGroupsConfigurationService: TutorialGroupsConfigurationService,
         private courseManagementService: CourseManagementService,
         private alertService: AlertService,
-        private courseCalculationService: CourseScoreCalculationService,
+        private courseStorageService: CourseStorageService,
 
         private cdr: ChangeDetectorRef,
     ) {}
 
+    course: Course;
```

Comment:
I'd revert this change and keep the course on the top with the other attributes.